microdnf/commands/advisory/advisory_summary.cpp

```diff
@@ -36,23 +39,82 @@ using namespace libdnf::cli;
 AdvisorySummaryCommand::AdvisorySummaryCommand(Command & parent) : AdvisorySummaryCommand(parent, "summary") {}
 
 
-AdvisorySummaryCommand::AdvisorySummaryCommand(Command & parent, const std::string & name) : Command(parent, name) {
+AdvisorySummaryCommand::AdvisorySummaryCommand(Command & parent, const std::string & name)
```

Comment:
I think it'd be better if all the advisory commands shared a base class (or there was another generic mechanism to share code) instead of one of them be designed as one.

Reply:
Changed in to a shared base class.
Let me know what you think, I might do that also for the repoinfo/repolist (or other) commands.